action.yml

```
@@ -315,6 +335,14 @@ runs:
         PATH=$PATH:$(pwd)
         cd $GITHUB_WORKSPACE
         digger
+
+    - uses: actions/cache/save@v4
+      name: cache-save
+      with:
+        path: ${{ github.workspace }}/cache
+        key: ${{ runner.os }}-${{ hashFiles('**/cache') }}
```

Over here we should add the hash, and also I suggest we use digger-xxxx as a convention

```suggestion
        path: ${{ github.workspace }}/cache
        key: digger-${{ hashFiles('**/cache') }}
```